lib/matplotlib/path.py

```diff
@@ -620,11 +623,20 @@ def get_extents(self, transform=None, **kwargs):
             self = transform.transform_path(self)
         if self.codes is None:
             xys = self.vertices
+        # Short cut the case with only straight lines
```

Comment:
Should be moved into the elif block. See the above comment for a possible wording.